File: lab6/TripAPI/TripAPI/DTOs/LinerDto.cs
using System.Text.Json.Serialization;

namespace CruiseCompany.DTOs;

public class LinerDto
{
    public int Id { get; set; }
    public string Name { get; set; } = null!;
    public int Capacity { get; set; }

    [JsonPropertyName("_class")]
    public string Class { get; set; } = null!;
    public int YearBuilt { get; set; }
    public List<RouteBriefDto> Routes { get; set; } = new();
}

File: lab6/app/app/src/main/java/com/example/myapp/data/dto/LinerDto.java
package com.example.myapp.data.dto;

import java.util.List;

public class LinerDto {
    public int id;
    public String name;
    public int capacity;
    public String _class; // if your JSON uses "class", map via @SerializedName
    public int yearBuilt;
    public List<RouteBriefDto> routes;
}
